scripts/update

```diff
@@ -7,64 +7,66 @@ fi
 
 DIR="$(dirname "$0")"
 
-
 function usage() {
-
     echo -n \
 "Usage: $(basename "$0")
-Setup external project dependencies.
+
+Update project dependencies, build assembly JARs, run database migrations, 
+and build static asset bundle.
 "
 }
 
 function run_database_migrations() {
     # Check if database migrations have already been initialized
     set +e
     docker-compose \
-        exec -T postgres gosu postgres psql -d rasterfoundry -c 'select 1 from __migrations__' &>/dev/null
```

Comment:
This check wasn't working. `run_database_migrations()` would try to run initialize migrations even if they already were initialized, and then the script would die. I applied a similar fix as in #4626. 

Not sure what the underlying problem is, but it may have to do with the new images we built in https://github.com/azavea/docker-postgis/pull/16.

Reply:
I would consider removing the `/dev/null` output redirect here too. If we run into issues later, they'll get surfaced better.